ansible/vars/dev.yml

```diff
@@ -9,7 +9,7 @@ ECRYPTFS_PASSWORD: 'xxBvYiMZ3YVIVkFR'
 KSPLICE_ACTIVE: no
 KSPLICE_ACTIVATION_KEY:
 
-AMQP_USER: hq_worker
+AMQP_USER: "{{ deploy_env }}_worker"
```

Comment:
Is this right? On prod is this `www_worker` and on staging it's `staging_worker` and on india it's `india_worker`?